rbac/management/group/view.py

```diff
@@ -797,8 +797,8 @@ def principals(self, request, uuid=None):
                     )
 
                 # Prettify the output payload and return it.
-                page = self.paginate_queryset(service_accounts)
-                serializer = PrincipalSerializer(page, many=True)
+                self.paginate_queryset(service_accounts)
```

Comment:
Do we still need to call this if we are not assigning it to a variable?

Reply:
first I wanted to answer like this ... yes we need this will set correctly the `self.paginator` and this is needed for `meta` and `links` to be created correctly in the `StandardResultsSetPagination` class `when self.get_paginated_response(service_accounts)` is called 2 lines below

then I realised how we handle user based principals in this endpoint and added a serializer for service account so logic for displaying correct data according limit and offset will be handled by rest-framework's serializer ... I added this change as fixup to be visible what I changed but I will rebase it after PR review ... let me please know, what you think

Reply:
Thanks Petra! I see what you did & it makes much more sense to me now.